guacamole/src/main/frontend/src/translations/de.json

```diff
@@ -446,21 +538,28 @@
         "FIELD_OPTION_SECURITY_RDP"   : "RDP Verschlüsselung",
         "FIELD_OPTION_SECURITY_TLS"   : "TLS Verschlüsselung",
 
-        "FIELD_OPTION_SERVER_LAYOUT_DA_DK_QWERTY": "Dänisch (Qwerty)",
```

Comment:
Any particular reason this was re-organized and moved down below? I think it should still be at the top?

Reply:
It should be on top yes.
But since it is down on `en.json` I aligned it to keep comparing easier.

If you want I move it up on both `en.json` and `de.json`

Reply:
Sure, that would be fine if you want to fix it in both locations.

Reply:
I fixed it in all langugaes where it was present.